stripe/deploy.gradle

```diff
@@ -1,14 +1,5 @@
-/*
- * Based on the example at Chris Banes's repository that allows signing
- * without manually creating a maven file.
- *
- * The original can be found at
- * https://raw.github.com/chrisbanes/gradle-mvn-push/master/gradle-mvn-push.gradle
- */
-
-def isReleaseBuild() {
```

Comment:
Why not keep this? It seems helpful for local testing, so you don't have to sign the release if it's a snapshot build.

Reply:
We don't use SNAPSHOT releases, so it seemed like extra noise.